### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rdrsadhu**
- [x] Brief overview of How help-activity works
- [x] How to edit Help and Contribute
- [x] How to translate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### How help-activity works:
+
```
Comment (godiard) [inline README.md]: Help Activity is also a activity itself, where yo u can get all the help content, Would be good have a paragraph about that.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+when contextual help( <kbd>alt</kbd> + <kbd>shift</kbd> + <kbd>h</kbd> ) is requested from within an sugar activity, the [helplink.json](helplink.json) file matches the bundle_id(the same as defined in the particular activity.info file) with a corresponding html page and renders its content in a tiny browser.
+
+Alternatively, all documentation is also available at https://help.sugarlabs.org/
+
+### How to edit Help and Contribute:
+https://help.sugarlabs.org/en/how_to_help.html
+
```
Comment (godiard) [inline README.md]: Probably is better link to the link in this repository https://github.com/godiard/help-activity/blob/master/source/how_to_help.rst

File: README.md (hunk 1 above)
Comment (rdrsadhu) [reply README.md]: added.

File: README.md (hunk 2 above)
Comment (rdrsadhu) [reply README.md]: changed.

**Comment by godiard** [comment]
Thanks

**Pull request merged**